### PR TITLE
[PLT-7534] Add color prop to Avatar

### PR DESCRIPTION
**Pull request by ericauv**
## Description
* Add a `color` prop to `<Avatar>`, defaults to "grey" and can be one of "grey" or "white"
## Notes
* required for https://github.com/riipen/platform-web/pull/4242
## Screenshots
Avatar with background white, others with default grey background
Screen Shot 2021-06-01 at 3 08 07 PM

### CODE REVIEW

**Review by jordanell — COMMENTED** [review]
@ericauv I'm not entirely convinced this is the way to go. We already have `classes` which can be set to something with a specific background color. Just curious why we want to make it an explicit prop with this PR.

**Comment by ericauv** [comment]
> @ericauv I'm not entirely convinced this is the way to go. We already have `classes` which can be set to something with a specific background color. Just curious why we want to make it an explicit prop with this PR.

Most of our other `ui` components have a color prop, I think this change makes sense because it makes the use of our ui components more consistent.
I think we should either always use the color prop to set background-color of our ui components, or always use classes --> I think using the color prop is an easier/cleaner dev experience

**Comment by jordanell** [comment]
@ericauv that's fair. I think we should call this one `backgroundColor` though to distinguish it from other foreground color props.

File: packages/riipen-ui/src/components/Avatar.jsx [hunk 1]
```diff
@@ -25,6 +25,11 @@ class Avatar extends React.Component {
      */
     classes: PropTypes.array,
 
+    /**
+     * The color of the background for the Avatar
+     */
+    backgroundColor: PropTypes.oneOf(["grey", "white"]),
```
Comment (jordanell) [inline packages/riipen-ui/src/components/Avatar.jsx:31]: Sorry I should have caught this earlier. When we use greys in our props, we have the standard of using the fully qualified grey like "grey200", "grey600", or "greyA400", so I think in this case, we should probably use `grey200`.

**Review by jordanell — CHANGES_REQUESTED** [review]

**Review by jordanell — APPROVED** [review]

**Pull request merged**